src/main/java/com/minecolonies/achievements/AchSizeSettlement.java

```diff
@@ -0,0 +1,29 @@
+package com.minecolonies.achievements;
+
+import net.minecraft.init.Items;
+
+/**
+ * 
+ * @author Isfirs
+ * @since 0.2
+ */
+public class AchSizeSettlement extends AbstractSizeAchievement
+{
+
+    public static final int size = 5;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) "size" is the name of a field in "AbstractSizeAchievement". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2387)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00115)